.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
I wonder if we *always* want to do this on push to master? A lot of the time I'd prefer to group a whole bunch of PRs in a single combined release.

Reply:
It always runs on master but that doesn't mean it creates a release on every push. It creates a PR like the ones we have now so you can do exactly that, if there are new changesets on master, the PR will be updated.

Reply:
Ah, of course, very good! Let's ship it then :-)